generation/core/generator.js

```diff
@@ -70,6 +70,9 @@ module.exports = function getGenerator({
       if (methodArg === null) {
         console.error(method);
       }
+      if (!supportedTypes.includes(methodArg.type)) {
```

Comment:
I think we have a separate map for this, which gives you the methods that could not be generated for each class due to which types.

Reply:
Right ! I'll take it off

Reply:
👍